content/Software Support/Installation/Add-libraries-to-Arduino-IDE.md

```diff
@@ -67,21 +59,23 @@ Learn how to add libraries to Arduino IDE.
 
 ---
 
-## Using installed libraries
+## Using installed libraries {#using-libraries}
 
 When the library has been installed you can do the following:
 
-* Open any included example sketches from _File > Examples > \<Library Name\>_ in the top menu bar. Note that all libraries do not come with examples.
-* Include the library in your sketch by selecting _Sketch > Include Library > \<Library Name\>_.
+- Open any included example sketches from _File > Examples > \<Library Name\>_ in the top menu bar.
+
+  > [!NOTE]
+  > Some libraries don't come with examples.
+
+- Include the library in your sketch by selecting _Sketch > Include Library > \<Library Name\>_.
 
 ---
 
 ## Further reading
 
-* [Uninstall a library from Arduino IDE](https://support.arduino.cc/hc/en-us/articles/360016077340-Uninstall-a-library-from-Arduino-IDE)
-
-* [Installing libraries in the Arduino IDE 1](https://docs.arduino.cc/software/ide-v1/tutorials/installing-libraries#importing-a-zip-library) on Arduino Docs
+- [Uninstall a library from Arduino IDE](https://support.arduino.cc/hc/en-us/articles/360016077340-Uninstall-a-library-from-Arduino-IDE)
 
-* [Installing libraries in the Arduino IDE 2](https://docs.arduino.cc/software/ide-v2/tutorials/ide-v2-installing-a-library) on Arduino Docs
+- [Installing libraries in the Arduino IDE 2](https://docs.arduino.cc/software/ide-v2/tutorials/ide-v2-installing-a-library) on Arduino Docs
```

Comment:
```suggestion
- [Installing libraries in the Arduino IDE](https://docs.arduino.cc/software/ide-v2/tutorials/ide-v2-installing-a-library) on Arduino Docs
```
For consistency since we're referring to it as IDE everywhere else.